README.md

```diff
@@ -18,10 +18,9 @@ Tool verifies the following scenarios to validate/troubleshoot DNS in EKS cluste
 - Verify coredns service (i.e. kube-dns) exist and its endpoints.
 - Performs DNS resolution against CoreDNS ClusterIP (e.g. 10.100.0.10) and individual Coredns pod IPs.
 - Detects if Node Local DNS cache is being used.
-- Verify that the inter-node communication is not blocked by a Security Group.
 - Verify EKS Cluster Security Group is configured correctly (Incorrect configs can prevent communication with coredns pods).
 - Verify Network Access Control List (NACL) rules are not blocking outbound TCP and UDP access on port 53 (which is required for DNS resolution).
-- Enable `log` plugin in Coredns Configmap for debugging and checks for errors in the Coredns pod logs
+- Checks for errors in the Coredns pod logs (Only If `log` plugin is enabled in Coredns Configmap).
```

Comment:
Now, script will not make any changes in cluster environment...everything is read only....
If log plugin is not enabled, it will skip checking errors in coredns pod logs.